### PR TITLE
Explicit conda-forge in docs

### PR DESCRIPTION
**Pull request by maddenp-noaa**
**Synopsis**

Try to make the docs a little more explicit and clearer w.r.t. conda use.

**Type**

- [x] Documentation

**Impact**

- [x] This is a non-breaking change (existing functionality continues to work as expected)

**Checklist**

- [x] I have added myself and any co-authors to the PR's _Assignees_ list.
- [x] I have reviewed the documentation and have made any updates necessitated by this change.

### CODE REVIEW

**Review by elcarpenterNOAA — APPROVED** [review]
LGTM.

File: docs/sections/contributor_guide/developer_setup.rst [hunk 1]
```diff
@@ -6,13 +6,13 @@ Creating a ``bash`` Development Shell
...
-.. include:: ../../shared/miniforge3_instructions.rst
+.. include:: ../../shared/miniforge_instructions.rst
 
-#. Install the :anaconda-condev:`condev package<>` into the base environment.
+#. Install the :anaconda-condev:`condev package<>` into the ``base`` environment.
 
    .. code-block:: text
 
-      conda install -y -c maddenp condev
+      conda install -y -c maddenp -c conda-forge --override-channels condev
```
Comment (christinaholtNOAA) [inline docs/sections/contributor_guide/developer_setup.rst:15]: Both `conda-forge` and `maddenp` here? Is that explicit for dependencies?

File: docs/sections/contributor_guide/developer_setup.rst [hunk 2]
```diff
@@ -26,23 +26,17 @@ If the above is successful, you will be in a ``uwtools`` development shell. See
 Future ``make devshell`` invocations will be almost instantaneous, as the underlying virtual environment will already exist. In general, all source code changes will be immediately live in the development shell, subject to execution, test, etc. But some changes --- especially to the contents of the ``recipe/`` directory or to the ``src/setup.py`` module --- may require recreation of the development shell. If you know this is needed, or when in doubt:
 
   #. Exit the development shell.
-  #. Run ``make clean-devenv`` (or ``conda env remove -n DEV-uwtools``) to remove the old environment.
+  #. Run ``make clean-devenv`` (or ``conda env remove -y -n DEV-uwtools``) to remove the old environment.
```
Comment (christinaholtNOAA) [inline docs/sections/contributor_guide/developer_setup.rst:29]: This one got me recently when we were working on MPAS.

File: docs/sections/user_guide/installation.rst [hunk 3]
```diff
@@ -5,7 +5,7 @@ Installation
 
    Developers should visit the :doc:`Developer Setup <../contributor_guide/developer_setup>` section located in the :doc:`Contributor Guide <../contributor_guide/index>`.
 
-The recommended installation mechanism uses the Python package and virtual-environment manager :conda:`conda<>`. Specifically, these instructions assume use of the :miniforge:`Miniforge<>` variant of :miniconda:`Miniconda<>`, built to use, by default, packages from the :conda-forge:`conda-forge<>` project. Users of the original :miniconda:`Miniconda<>` or the :anaconda:`Anaconda distribution<>` should add the flags ``-c conda-forge --override-channels`` to ``conda`` commands to specify the required package channels.
+The recommended installation mechanism uses the Python package and virtual-environment manager :conda:`conda<>`. The :miniforge:`Miniforge<>` variant of :miniconda:`Miniconda<>`, which by default packages from the :conda-forge:`conda-forge<>` project, is an especially useful basis for working with conda.
```
Comment (christinaholtNOAA) [inline docs/sections/user_guide/installation.rst:8]:
```suggestion
The recommended installation mechanism uses the Python package and virtual-environment manager :conda:`conda<>`. The :miniforge:`Miniforge<>` variant of :miniconda:`Miniconda<>`, which by default uses packages from the :conda-forge:`conda-forge<>` project, is an especially useful basis for working with conda.
```

**Review by christinaholtNOAA — APPROVED** [review]
LGTM!

**Review by maddenp-noaa — COMMENTED** [review]
Sorry, I never submitted my comments! I'll hold off on merging until later today just in case you want to read them.

**Comment by christinaholtNOAA** [comment]
Thanks for the additional context!

**Pull request merged**